style.css

```diff
@@ -0,0 +1,20 @@
+section {
```

Comment:
Please it more meaningful to start styling the whole page using <code>*</code> where you can remove <code>padding</code> and <code>margin</code> come with some tag and set them to 0

Reply:
Can also add the <code>box-sizing</code> and set it to <code>border-box</code> to avoid the width and height sumation